pkg/gardenlet/controller/shoot/status/reconciler.go

```diff
@@ -178,6 +194,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 }
 
 func needsReconcile(shoot *gardencorev1beta1.Shoot) bool {
+	if kubernetesutils.HasMetaDataAnnotation(shoot, v1beta1constants.GardenerOperation, v1beta1constants.GardenerOperationReconcile) {
+		return false
+	}
```

Comment:
This annotation is never present because it is automagically removed by `gardener-apiserver`. What am I missing here?

Reply:
You are right. Will remove it.